operators/src/pro/cache/tile_cache.rs

```diff
@@ -714,13 +926,18 @@ mod tests {
     #[tokio::test]
     async fn it_evicts_lru() {
         // Create cache entry and landing zone entry to geht the size of both
+        //let op0 = op(0);
```

Comment:
can be removed?

Reply:
that depends. If we account for the "OperatorCache" then we need that.

Reply:
a285a865bc1432ceaa236ad0f1749031b9b5dc38